src/wbase/UberJobData.cc

```diff
@@ -258,18 +255,26 @@ void UJTransmitCmd::action(util::CmdData* data) {
     bool transmitSuccess = false;
     try {
         json const response = client.readAsJson();
-        if (0 != response.at("success").get<int>()) {
+        auto respMsg = protojson::ResponseMsg::createFromJson(response);
+        if (respMsg->success) {
             transmitSuccess = true;
+            string note = response.at("note");
+            if (note != "") {
+                LOGS(_log, LOG_LVL_INFO, response);
+            }
         } else {
-            LOGS(_log, LOG_LVL_WARN, cName(__func__) << " Transmit success == 0");
+            LOGS(_log, LOG_LVL_WARN, cName(__func__) << " Transmit success == 0 " << response);
```

Comment:
The way this logging statement is written would inject some unconventional (and hard to parse) text into the log stream:
```
Transmit success == 0 
``` 
Perhaps you want to see this instead of the C/C++ comparison operator?
```
Transmit success=0 
```